Employees/Models/Employee.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Employees.Models
+{
+    public abstract class Employee
+    {
+        private readonly string name;
+        private readonly double salary;
+
+        public Employee(string name, double salary)
+        {
+            this.name = name;
+            this.salary = salary;
+        }
+
+        public string Name => name;
+
+        public virtual double Salary => salary;
+
+    }
+    public abstract class EmployeeWithManager : Employee
```

Comment:
no need for this to be an empty class.
replace it with an interface IAmManaged for example and use the interface instead